ndsl/grid/generation.py

```diff
@@ -297,12 +297,34 @@ def __init__(
         self._dy_center = None
         self._area = None
         self._area_c = None
-        (
-            self._ks,
-            self._ptop,
-            self._ak,
-            self._bk,
-        ) = self._set_hybrid_pressure_coefficients(eta_file, ak, bk)
+        if eta_file != "None":
```

Comment:
Do we prefer `is not None` to `!- None`?

Reply:
Also should we set the default vaule of eta_file to None instead of "None"?

Reply:
That makes more sense I will push that shortly.